src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -828,4 +850,41 @@ public static int ConvertScaleFactorToUI(int scaleValue)
            return (scaleValue / 2) + 1;
         }
     }
+
+    /// <summary>
+    /// This class represent a Tab and is used for store just one Expander info(due that just one Expander can be expanded at one time)
+    /// </summary>
+    public class TabSettings : INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged;
+        private void OnPropertyChanged(string propertyName)
+        {
+            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+        }
+
+        /// <summary>
+        /// Tab Name (e.g. Features o Visual Settings)
```

Comment:
`or`?

Reply:
@QilongTang fixed in commit: https://github.com/DynamoDS/Dynamo/pull/11689/commits/5ee62bb8dcb52c07eef3929916511372bdbcd128